data_structures/linked_list/add_nodes.py

```diff
@@ -0,0 +1,81 @@
+"""Adding two integers of same length using Linked lists
+
+Example add 321 + 248 = 569
+
+(1)->(2)->(3) + (8)->(4)->(2) = (5)->(6)->(9) """
+
+
+# node definition
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+        # self.head=None
+
+
+# linkedlist definition
+class LL:
+    def __init__(self):
+        self.head = None
+
+    # method to print the linked list
+    def printLL(self):
```

Comment:
It is more Pythonic to define a [.__str__()](https://docs.python.org/3/reference/datamodel.html#object.__str__) method that returns a str then then we can do `print(my_linked_list)`.